pcap_ex.c

```diff
@@ -290,7 +290,7 @@ pcap_ex_compile_nopcap(int snaplen, int dlt, struct bpf_program *fp, char *str,
 	char ebuf[PCAP_ERRBUF_SIZE];
 	int ret = -1;
 	
-	_mktemp(path);
+	mktemp(path);
```

Comment:
...presumably bringing back the complaints.  Presumably they're just annoying warnings (if you want to eliminate them, arrange, somehow, to call `mktemp()`/`unlink()` on UN\*X and `_mktemp()`/`_unlink()` on Windows, or, for unlinking, use `remove()`, if you want to require an ANSI/ISO C environment).